<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-10-13 17:34:11.743 ***</b>
=ERROR REPORT==== 13-Oct-2024::17:34:11.742841 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244623.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1973}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1934}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-10-13 17:34:11.743 ***</b>
=CRASH REPORT==== 13-Oct-2024::17:34:11.743331 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244631.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2426)
    ancestors: [<0.244623.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244623.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-10-13 17:34:51.572 ***</b>
=ERROR REPORT==== 13-Oct-2024::17:34:51.572209 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248975.0&gt;,
                               {#Ref&lt;0.4268240406.1237843969.152903&gt;,1007427,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.4268240406.1237843969.152903&gt;,1007427,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-10-13 17:34:51.572 ***</b>
=CRASH REPORT==== 13-Oct-2024::17:34:51.572664 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248994.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.4268240406.1237843969.152903&gt;,1007427,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [&lt;0.248975.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4505
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-10-13 17:39:27.214 ***</b>
=CRASH REPORT==== 13-Oct-2024::17:39:27.214087 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254794.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.4268240406.1239154689.212167&gt;,6382,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1694)
    ancestors: [&lt;0.254792.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28121
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-13 17:39:36.405 ***</b>
=ERROR REPORT==== 13-Oct-2024::17:39:36.405137 ===
** Generic server <0.256092.0> terminating 
** Last message in was {nodedown,'peer_down_crash-459619-11743@cf893845472b'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-459619-11743",
                              peer_down => crash},
                            'peer_down_crash-459619-11743@cf893845472b',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,49,46,49,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-459619-11743",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQGNmODkzODQ1NDcyYgAD6FwAAAAAZwvmKg=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-459619-11743@cf893845472b'}},
                            {<0.256059.0>,#Ref<0.4268240406.1232338946.35345>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-459619-11743@cf893845472b'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-10-13 17:39:36.406 ***</b>
=CRASH REPORT==== 13-Oct-2024::17:39:36.405879 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256092.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-459619-11743@cf893845472b'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [<0.256059.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 11231
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-13 17:39:37.260 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:36.258914 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-511490-11743@cf893845472b' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-13 17:39:37.746 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:37.746769 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-385445-11743@cf893845472b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-13 17:39:37.758 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:37.758700 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-385317-11743@cf893845472b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-13 17:39:53.408 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:52.407012 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-512290-11743@cf893845472b' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-13 17:39:53.913 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:53.913793 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-512322-11743@cf893845472b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-13 17:39:53.929 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:53.929765 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-397732-11743@cf893845472b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-13 17:39:56.069 ***</b>
=ERROR REPORT==== 13-Oct-2024::17:39:56.069098 ===
** Generic server <0.256439.0> terminating 
** Last message in was {'EXIT',<0.256407.0>,
                               {#Ref<0.4268240406.1231552516.82046>,1573217,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-398244-11743",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-398244-11743@cf893845472b',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,49,46,49,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-398244-11743",
                                     "-user","peer"],
                                    #Port<0.3119>,undefined,<<>>,running,
                                    {<0.256407.0>,
                                     #Ref<0.4268240406.1231552515.171684>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.4268240406.1231552516.82046>,1573217,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-13 17:39:56.069 ***</b>
=CRASH REPORT==== 13-Oct-2024::17:39:56.069625 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256439.0>
    registered_name: []
    exception exit: {#Ref<0.4268240406.1231552516.82046>,1573217,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256407.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3119>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 13726
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-13 17:39:56.649 ***</b>
=ERROR REPORT==== 13-Oct-2024::17:39:56.649541 ===
** Generic server <0.256465.0> terminating 
** Last message in was {'EXIT',<0.256405.0>,
                           {#Ref<0.4268240406.1231552515.171400>,2153706,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "ZBVZBWMKYLHZZANKJKQC","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@cf893845472b.2024-10-13_17.24.25/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-385989-11743",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@cf893845472b.2024-10-13_17.24.25/make_test_dir.stdlib_test.logs/run.2024-10-13_17.24.59/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-385989-11743",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.4/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-385989-11743@cf893845472b',
                                    "/usr/local/lib/erlang-26.2.5.4/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-385989-11743",
                                     "-setcookie","ZBVZBWMKYLHZZANKJKQC",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@cf893845472b.2024-10-13_17.24.25/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-385989-11743",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@cf893845472b.2024-10-13_17.24.25/make_test_dir.stdlib_test.logs/run.2024-10-13_17.24.59/log_private/",
                                     "-user","peer"],
                                    #Port<0.3145>,undefined,<<>>,running,
                                    {<0.256405.0>,
                                     #Ref<0.4268240406.1231552516.83136>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.4268240406.1231552515.171400>,2153706,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-13 17:39:56.650 ***</b>
=CRASH REPORT==== 13-Oct-2024::17:39:56.650352 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256465.0>
    registered_name: []
    exception exit: {#Ref<0.4268240406.1231552515.171400>,2153706,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256405.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3145>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 10958
    stack_size: 29
    reductions: 20921
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-13 17:39:56.979 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:55.977817 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-512546-11743@cf893845472b' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-13 17:39:57.178 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:57.178777 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-385925-11743@cf893845472b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-13 17:39:57.337 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:57.337819 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-386021-11743@cf893845472b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-13 17:39:57.543 ***</b>
=WARNING REPORT==== 13-Oct-2024::17:39:57.543774 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-512610-11743@cf893845472b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-13 18:01:06.686 ***</b>
=ERROR REPORT==== 13-Oct-2024::18:01:06.686129 ===
** Generic server <0.333292.0> terminating 
** Last message in was {'EXIT',<0.333289.0>,
                               {#Ref<0.4268240406.1268514817.147161>,351301,
                                [{child1,<0.333298.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.333292.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.333298.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458552],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.4268240406.1268514817.147161>,351301,
    [{child1,<0.333298.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-13 18:01:06.686 ***</b>
=ERROR REPORT==== 13-Oct-2024::18:01:06.686135 ===
** Generic server <0.333291.0> terminating 
** Last message in was {'EXIT',<0.333289.0>,
                               {#Ref<0.4268240406.1268514817.147161>,351301,
                                [{child1,<0.333298.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.333291.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.333297.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458552],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.4268240406.1268514817.147161>,351301,
    [{child1,<0.333298.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-13 18:01:06.686 ***</b>
=CRASH REPORT==== 13-Oct-2024::18:01:06.686589 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.333292.0>
    registered_name: []
    exception exit: {#Ref<0.4268240406.1268514817.147161>,351301,
                     [{child1,<0.333298.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.333289.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14920
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-13 18:01:06.687 ***</b>
=CRASH REPORT==== 13-Oct-2024::18:01:06.686817 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.333291.0>
    registered_name: []
    exception exit: {#Ref<0.4268240406.1268514817.147161>,351301,
                     [{child1,<0.333298.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.333289.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 14993
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2024-10-13 18:01:50.189 ***</b>
=CRASH REPORT==== 13-Oct-2024::18:01:50.189183 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.433449.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [bridge9212,<0.433444.0>]
    message_queue_len: 0
    messages: []
    links: [<0.433450.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4062
  neighbours:
    neighbour:
      pid: <0.433450.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.433449.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during zzz_SUITE:lc_graph/1 2024-10-13 18:27:59.581 ***</b>
=WARNING REPORT==== 13-Oct-2024::18:27:59.578547 ===
Process <0.436930.0> on node 'test_server@cf893845472b' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Sun Oct 13 2024 17:24:59<!--/date--><br />
</div>
</center>
</body>
</html>
